.github/steps/2-tbd.md

```diff
@@ -89,19 +90,19 @@ Now that you have enabled secret scanning push protection, certain new secrets w
 
 5. Delete `<REMOVEME>` from the string you just pasted. The `<REMOVEME>` string is there so secret scanning doesn't create an alert before you're able to test push protection. Your file should look like this:
 
-    ![A screenshot of credentials.yml being edited in the GitHub web UI. A newly added github-token is highlighted.](/images/push-protection.png)
+    ![Screenshot of credentials.yml being edited in the GitHub web interface. A newly added github-token is highlighted.](/images/push-protection.png)
 
-6.  Select **Commit changes...**
-7.  Select **Commit changes**
-8.  At this point, an alert will show on your page informing you that a new secret is being added to the repository.
+6.  Select **Commit changes...**.
+7.  Select **Commit changes**.
+8.  Instead of committing the updated file to your repository, a push protection alert warns you that your changes include a GitHub Personal Access Token.
 
 ### :keyboard: Activity 3.3: Bypass push protection
 
-Now that you're aware of the secret in your commit, you should remove the secret from the commit and commit history, then attempt the push again. In some cases, you may be willing to accept the risk of adding a secret to your repository. In those situations, you can choose to bypass push protection. In this activity, you will bypass push protection and write the token to your repository (don't worry, the example token is safe).
+Now that you're aware of the secret in your changes, you should remove the secret and then attempt the commit again. In some cases, you may be willing to accept the risk of adding a secret to your repository. In those situations, you can choose to bypass push protection. In this activity, you will bypass push protection and write the token to your repository (don't worry, the example token is safe).
```

Comment:
I don't think we should be mentioning editing the commit history in this section since the UI blocks us from committing the secret.

I think this behaves differently if you're working locally or in a codespace, but here the commit history is not relevant.

Reply:
I suggest that we add a note for people using a local environment or codespace, to explain that they will also need to revise their commit history. We can direct them to the related Skills course: https://github.com/skills/change-commit-history.

Reply:
See lines 92-93 above.